### DISCUSSION TITLE
Webview: cannot open phone links in webview (tel:+386...)

### DISCUSSION THREAD

**Discussion by easydrivesi** [inspireui/support#5685] [Working on Features]
Bravo for fixing browse files and upload in webview!!!

But now there's another problem in Android webview. When there's a phone link (tel:+386...), webview doesn't open a dialer like before, but shows this: 
![Screenshot_20221201_163335_com easydrive voznik_edit_182529267725793](https://user-images.githubusercontent.com/102716951/205095068-30ff6794-b88c-48bd-9625-265baad1fed8.jpg)

**Comment by simonpham** [comment]
This feature is available in FluxStore 3.4.0
